http/headers/feature-policy.json

```
@@ -1853,6 +1853,56 @@
             }
           }
         },
+        "web-share": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/web-share",
+            "support": {
+              "chrome": {
+                "version_added": null
```

This appears to have not yet shipped in Chromium: https://www.chromestatus.com/feature/6362499966304256. I'd apply this change to all the Chromium browsers (Chrome, Edge, Opera, Samsung Internet, and WebView)

```suggestion
                "version_added": false
```

---

Fixed (and rebased). Thank you!